code/modules/paperwork/paper_bundle.dm

```diff
@@ -11,9 +11,13 @@
 	layer = 4
 	pressure_resistance = 2
 	attack_verb = list("bapped")
-	var/amount = 0 //Amount of items clipped to the paper. Note: If you have 2 paper, this should be 1
+	var/amount = 0 //Amount of total items clipped to the paper. Note: If you have 2 paper, this should be 1
+	var/photos = 0 //Amount of photos clipped to the paper.
 	var/page = 1
 	var/screen = 0
+	gender = NEUTER
```

Comment:
why are you setting gender here? 
also you're overriding a previous definition on L3 `gender = PLURAL`

Reply:
Gender needs to be neutral for grammar with the examine, also overriding gender = PLURAL is just an oversight on my part. Since the base atom gender is neuter I should probably just remove both.
Plural: ![image](https://user-images.githubusercontent.com/91113370/181885149-d4cf231d-0253-4657-b91c-e85736602f0c.png)
Neuter: ![image](https://user-images.githubusercontent.com/91113370/181886265-90ec3426-8e3d-40d1-9299-f505f74db626.png)